src/borrower/feed/navfeed.sol

```diff
@@ -217,23 +242,15 @@ abstract contract NAVFeed is Auth, Discounting {
 
         uint totalDiscount = rmul(latestDiscount, rpow(discountRate.value, safeSub(nnow, nLastUpdate), ONE));
 
-        // Loop over the loans which matured in between the last NAV update and now.
-        // Then remove their discounted future value from the total discount as they are overdue.
-        uint diff = 0;
-        for(uint i = nLastUpdate; i < nnow; i = i + 1 days) {
```

Comment:
We still need to calculate the error and subtract it. 
I think here we could also calculate the new `overdueButNotWrittenOff`  and add it again, because we anyway need to iterate over the buckets from the lastUpdate until now.